File: Practice/11/Python/11.py
import math

a=float(input())
b=int(input())

c=1
c=a
i=1

while (i<abs(b)):
    c*=a
    i+=1

if b>0:
    print(c)
elif b<0:
    print(1/c)
elif b==0:
    print("Результат = 1")

File: Practice/11/C++/11/11/11.cpp
#include <iostream>
#include<math.h>
using namespace std;
int main()
{

    double a, b;
    int x, i = 1;

    cin >> a >> x;

    b = a;
    while (i < abs(x)) {
        a = a * b;
        i++;
    }
    if (x > 0) {
        cout << a << endl;
        return 0;
    }
    else if (x < 0) {
        cout << 1 / a << endl;
        return 0;
    }
    else if (x == 0) {
        cout << 1 << endl;
        return 0;
    }
}